docs/docs/integrations/providers/cohere.mdx

```diff
@@ -46,6 +46,55 @@ print(llm.invoke("Come up with a pet name"))
 ```
 
 Usage of the Cohere (legacy) [LLM model](/docs/integrations/llms/cohere) 
+
+### Tool calling
```

Comment:
could we add this to the chat model doc instead docs/docs/integrations/chat/cohere.ipynb

Reply:
hey @baskaryan, is there any downside to add it in both places?

Reply:
Hey @baskaryan I've added it to both the chat model doc as well as provider. Do lmk if there are any concerns in doing that.